charts/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $namespace := .Values.common.namespace | required "A valid .common.namespace is required!" -}}
```

Comment:
This should be deleted, namespace is automatically overriden when users install it in a specific namespace with the `--namespace` flag

Reply:
removed explicit namespace definition in the chart.